caput/misc.py

```diff
@@ -1,11 +1,15 @@
 """A set of miscellaneous routines that don't really fit anywhere more specific."""
```

Comment:
Can you review https://github.com/radiocosmology/caput/pull/164 so we can merge that before making changes to `caput.misc`?